src/guides/v2.3/ext-best-practices/tutorials/dynamic-row-system-config.md

```diff
@@ -171,7 +171,67 @@ class TaxColumn extends Select
 
 This block sets values for the drop-down option.
 
-## Step 4: Clean cache
+## Step 4: Set default values - **OPTIONAL**
+It is possible to set defaults for a dynamic row configuration, this is done by adding additional XML to the defaults block in the `config.xml` file for the module.
+
+Add a block to the `<default>` section in the file as it would be done normally but do not add any values. For the example this tutorial has been using, this would look like :
```

Comment:
I recommend to cut the fluff. Less chance of deviating from the tutorial at hand.
Also, it doesn't hurt to reiterate the filename in case they skimmed over the previous paragraph and didn't quite catch that.

```suggestion
Add a block to the `<default>` section of the `config.xml` file and do not include any values:
```

Reply:
Fluff - and I thought I was being nice and clear 😄